lib/turn.rb

```diff
@@ -0,0 +1,20 @@
+class Turn
+  attr_reader :card, :guess 
+
+  def initialize(guess, card)
+    @guess = guess
+    @card = card
+  end
+
+  def correct?
+    guess == @card.answer
```

Comment:
I see you're using the `@` to reference instance variables interchangeably - my advice is to **not** do that; if you're going to reference, access, or re-assign any instance variables, do so with the `@`. Otherwise it can get confusing and inconsistent with what you're trying to reference.

Reply:
In other words, this should look like 

`@guess == @card.answer`

That way, we know instantly that `@guess` refers to line 5, and not a method somewhere else.